geostore/check.py

```diff
@@ -13,3 +13,5 @@ class Check(Enum):
     STAGING_ACCESS = "staging bucket access"
     UNKNOWN_CLIENT_ERROR = "unknown client error"
     UNKNOWN_MULTIHASH_ERROR = "unknown multihash error"
+    INVALID_STAC_ROOT_TYPE = "Root type should be catalog or collection"
```

Comment:
The value should be formatted like the other ones, so it shouldn't be capitalised. By the way, can you please re-sort the enums so they're still alphabetical?

Reply:
The list isn't quite sorted yet; `NO_ASSETS_IN_DATASET` should be in the middle. (This is why I recommend leaving the original commenter to resolve conversations - it ensures that each comment is verified as addressed by the commenter.)

Reply:
Updated. Thanks.